data/json/recipes/food/brewing.json

```diff
@@ -0,0 +1,236 @@
+[
+    {
+        "type" : "recipe",
+        "result": "brew_milk_curdled",
+        "result_mult" : 18,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "cooking",
+        "difficulty": 5,
+        "skills_required": [ "survival", 4 ],
+        "time": 30000,
+        "batch_time_factors": [50, 4],
+        "autolearn": true,
+        "book_learn": [["survival_book", 3] , ["textbook_survival", 3], ["manual_survival", 3]],
+        "components": [
+            [ [ "milk", 15 ] ],
+            [ [ "vinegar", 3 ] ],
+            [
+                [ "wild_herbs", 40 ],
+                [ "stomach_large", 1 ],
+                [ "hstomach_large", 1 ],
+                [ "stomach", 2 ],
+                [ "hstomach", 2 ]
+            ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "brew_whiskey",
+        "result_mult" : 3,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
+        "skill_used": "cooking",
+        "difficulty": 5,
+        "time": 25000,
+        "batch_time_factors": [50, 4],
+        "book_learn": [ ["brewing_cookbook", 5], [ "scots_cookbook", 5 ] ],
+        "qualities":[ { "id": "DISTILL", "level": 2 } ],
+        "tools": [ [ [ "hotplate", 20 ], [ "char_smoker", 20 ], [ "toolset", 20 ], [ "fire", -1 ] ] ],
+        "components": [
+            [ [ "water", 3 ], [ "water_clean", 3 ] ],
+            [ [ "corn", 2 ], [ "irradiated_corn", 2 ], [ "cornmeal", 6 ], [ "barley", 1 ], [ "buckwheat", 1 ] ],
+            [ [ "sugar", 50 ] ],
+            [ [ "yeast", 1 ] ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "brew_vodka",
+        "result_mult" : 3,
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
+        "skill_used": "cooking",
+        "difficulty": 5,
+        "time": 50000,
+        "batch_time_factors": [50, 4],
+        "autolearn": [ [ "cooking", 6 ] ],
+        "book_learn": [ ["brewing_cookbook", 6] ],
+        "qualities":[ { "id": "DISTILL", "level": 2 } ],
```

Comment:
This shouldn't require `DISTILL`, that's only required by the later distillation stage that turns this into `vodka`